consensus/vbft/service.go

```diff
@@ -1060,10 +1060,6 @@ func (self *Server) onConsensusMsg(peerIdx uint32, msg ConsensusMsg, msgHash com
 			return
 		}
 		msgBlkNum := pMsg.GetBlockNum()
-		if msgBlkNum > self.GetCurrentBlockNo() {
```

Comment:
no larger than 10 to avoid attack?

Reply:
before processing, consensus msg has been verified from valid consensus peer with signatures in msg.